index.html

```diff
@@ -285,7 +433,7 @@ <h1>
     </section>
 </div>
 <div>
-   <section class="contact-form">
+   <section class="contact-form" id="contact-section">
```

Comment:
[OPTIONAL] Contact section has very little width on mobile. Please see the screenshot below and make sure that this section is similar to the design.
![contact](https://user-images.githubusercontent.com/35504149/130266313-a671843e-6a86-4e74-bf33-b2873b7eee6c.png)